app/views/firms/partials/_firm_details.html.erb

```diff
@@ -96,3 +96,17 @@
     </div>
   </div>
 </div>
+
+<div class="l-firm__row">
```

Comment:
Yeah it's definitely fine to do with an `l-*` because that's just saying that in this specific layout for this component, I'd like a visual divider.

I think the way you have it is fine, probably not worth going round the houses trying to work it out if things aren't the right width etc